src/Microsoft.AspNetCore.Http.Features/IHttpRequestFeature.cs

```diff
@@ -49,6 +49,11 @@ public interface IHttpRequestFeature
         string QueryString { get; set; }
 
         /// <summary>
+        /// The request target as it was sent in the HTTP request.
+        /// </summary>
+        string RawTarget { get; set; }
```

Comment:
I like `RawPath`, it's "hidden" already on the feature and people aren't stupid, let's not try to protect them?